src/api/app/views/webui2/webui/package/_side_links.html.haml

```diff
@@ -1,24 +1,25 @@
--# FIXME: refactor this file
 %ul.list-unstyled
   - if failures > 0
-    = render partial: 'show_failures', locals: { failures: failures, package_name: package.name, project: project }
+    = render partial: partial_path('side_links', 'show_failures'), locals: { failures: failures, package_name: package.name, project: project }
```

Comment:
I do take issue with this. I know what `render partial: 'directory/subdirectory/file'` does, I have to loop up the meaning and implementation of `partial_path`. It's code we do not need, which is the worst kind of code. Sorry...

Reply:
TL;DR
my bikeshed is green :alien:

actually, as soon as we get rid of ```webui/webui2``` is easier to change in one place than switch it  everywhere, we could put it in a variable, but that's exactly why helpers are there @hennevogel

Reply:
But once we switched to webui2 we would drop the `partial_path` or not? And is this planned to be used only for package partials or others as well?

Reply:
I would use it as well in the #5867 (huge paths are coming XD) but I removed it already. Thank you @bgeuken <3

Reply:
> my bikeshed is green

If I would think this is not important I would mark my comment as nitpick. I haven't :cowboy_hat_face: